RAOricutron/RAIntegration/.clang-format

```diff
@@ -0,0 +1,66 @@
+﻿DisableFormat: false
```

Comment:
it appears that your nested submodule was committed as raw code.

Reply:
Wow, that's an old mistake.